### PR TITLE
Reddit App blog

### PR DESCRIPTION
**Pull request by bmuhamood**

### CODE REVIEW

**Comment by bmuhamood** [comment]
- Completed BUILDING WITH ACTIVE RECORD. 
- Created Data model 
- Played with Associations
- Added in Commenting from the console

**Comment by BrittanyBlake** [comment]
### Status: Invalid 🛑 
****
Hi Team. This pull request has been marked as invalid because you did not follow the proper [Git flow](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59882084/). 

- You should be working in the` feature branch` of your project, and open a pull request to merge from the `feature branch` into the `development branch`.
- You are currently merging from your development branch into your feature branch.
****
Please make the required adjustments and submit a new code review request.

### Good Luck and Happy Coding 🍀 👩🏾‍💻

**Comment by bmuhamood** [comment]
Hi BrittanyBlake, 

Thanks a lot for the review. 
Actually I created develop and production branches,  **develop** branch to act as **feature branch** and merge to **production branch** once the project is approved. 

I made Production branch as a default branch. 

Thanks a lot
Best regards. 
Bbosa Muhamood

**Review by christianotieno — CHANGES_REQUESTED** [review]
# Status: Invalid CRR ⚠️ 
Hello, @bmuhamood thank you for the explanations put forward. 

But just as @BrittanyBlake stated out earlier, the project has not followed. 

- You are required strictly work with Git Flow: production `master` branch + main `development` branch + Pull Requests from feature branches to the `develop` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/git_flow_requirement.png) from Pathwright), which is not the case in this instance. You have mixed up the baches and it is hard to follow the flow with the different branch names. 


- Please take a step back and watch the video here [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59882084/) lesson.

- Then create a brand new repo with correct setup and feature branch (with DESCRIPTIVE name) and copy your code there or close this pull request and create another one with a correct feature branch.

###### This Code Review Request will be marked as `invalid` in your `Dashboard`, so please submit a new one once you are ready!

### Happy coding 💻 
#### Christian Otieno (TSE)
